backend/tournesol/serializers.py

```diff
@@ -91,8 +101,15 @@ class Meta:
             "language",
             "rating_n_ratings",
             "rating_n_contributors",
+            "duration",
         ]
 
+    # Convert duration to seconds to facilitate use of Humanize package
+    def get_duration(self, obj):
+        if obj.duration:
+            return int(obj.duration.total_seconds())
+        return 0
```

Comment:
@sandre35 @GresilleSiffle @amatissart 

There is some code duplication between `VideoSerializer` and `VideoSerializerWithCriteria`


Could this be refactored into 

```
class VideoSerializerWithCriteria(VideoSerializer):
    criteria_scores = VideoCriteriaScoreSerializer(many=True)
    
    class Meta(VideoSerializer.Meta):
        fields = VideoSerializer.Meta.fields + ['criteria_scores'] 
```